### ISSUE TITLE
Build from source

### ISSUE THREAD

**Issue by dougbtv** [redhat-nfvpe/kube-ansible#8]
Add option to do an install from source

**Comment by leifmadsen** [comment]
Talking about this, would be great to account for ability to point at another repository, and branch. By doing this, you could add your own changes against a Kubernetes version, and then test something specific before submitting a change upstream.

**Comment by leifmadsen** [comment]
Hrmmm, the more I think about this, the more I think we should almost avoid building from source directly out of this repository. There must be some sort of other "things" we can leverage from upstream that would allow us to build from source and consume it, similar to how Kubernetes is building.

Turns out, I found this thing called Planter that might solve this. We could potentially wrap it and consume the results in our deployments.

See https://github.com/kubernetes/test-infra/tree/master/planter
